Userland/Libraries/LibWeb/DOM/Document.h

```diff
@@ -642,7 +642,7 @@ class Document
     QuirksMode m_quirks_mode { QuirksMode::No };
 
     // https://dom.spec.whatwg.org/#concept-document-type
-    Type m_type { Type::HTML };
+    Type m_type { Type::XML };
```

Comment:
Please state in the commit message _why_ you're changing this, not _what_ you're changing (which is already obvious from the diff).

Reply:
I've updated the commit message to better explain why I made the change.

tl;dr: because the spec says so.